shared/components/ScheduleItem/index.js

```diff
@@ -1,20 +1,35 @@
 import React, { PropTypes } from 'react';
-import { formatDate } from '../../utilities/date';
 
-const ScheduleItem = ({
-  datetime,
-  text,
-}) => (
-  <li className="ScheduleItem" data-time={formatDate(datetime, 'HH:mm')}>
-    <p className="ScheduleItem__text">
-      {text}
-    </p>
-  </li>
+import ScheduleDetail from '../ScheduleDetail';
+
+const ScheduleItem = ({ item }) => (
+  <div className="ScheduleItem">
+    <div className="ScheduleItem__header">
+      <time className="ScheduleItem__header__time">
+        {item.time}
+      </time>
+      <h3 className="ScheduleItem__header__title">
+        {item.title}
+      </h3>
+    </div>
+    <div className="ScheduleItem__details">
+      <p className="ScheduleItem__details__description">
+        {item.description}
+      </p>
+      <ul className="ScheduleItem__details__list">
+        {item.details.map((detail, index) => <ScheduleDetail key={index} detail={detail} />)}
```

Comment:
Maybe move the `<li>` out of the `ScheduleDetail` component and into here to keep it near to the `<ul>`

    {item.details.map((detail, index) => <li><ScheduleDetail key={index} detail={detail} /></li>)}